docs/user-guide/command-line-interface.md

```diff
@@ -147,6 +148,10 @@ Examples:
 
 This option allows you to specify a parser to be used by eslint. By default, `espree` will be used.
 
+#### `--parser-options`
+
+This option allows you to specify parser options to be used by eslint.
```

Comment:
Could you add an example on how to configure parser options through command line?